text/0042-launcher-api-v0.6/api/appendable_data.md

```diff
@@ -39,39 +37,71 @@ Authorization: Bearer <TOKEN>
 #### Body
 |Field|Description|
 |-----|-----------|
+| name | base64 string representing [u8; 32] array |
 | isPrivate | Boolean. Optional Defaults to false |
-| filterType | Enum value. WHITE_LIST, BLACK_LIST |
-| filterKey | List of public keys. Public key as base64 string. Optional value defaults to empty list |
+| filterType | Enum value. WHITE_LIST, BLACK_LIST. Defaults to BLACK_LIST |
+| filterKey | List of signing key handles. Optional value defaults to empty list |
 
 ```
 {
+  name: base64 string,
   isPrivate: Boolean,
   filterType: Enum,
-  filterKeys: list of publickeys
+  filterKeys: list of signing key handles
 }
 ```
 
 ### Response
 
 #### Status Code
- ```
- 200
- ```
+```
+200
+```
+
+#### Body
+```
+{
+  handleId:u64 number representing the appendable data handle id
+}
+```
+
+## Save AppendableData
+
```

Comment:
I think we should add a text similar to what you added in Structured Data. something like this:

> The safe_core calls PUT to store the data in the network and POST to update an existing data in the network. When an appendable data is created for the first time, the application should call the PUT endpoint, and while updating an appendable data, the application should call the POST endpoint.